snuba/pipeline/stages/query_processing.py

```diff
@@ -34,10 +34,10 @@ def _process_data(
         if translated_storage_query:
             return translated_storage_query
 
+        run_entity_validators(cast(EntityQuery, query), pipe_input.query_settings)
```

Comment:
Before this change, entity validation was done only if the query was `LogicalQuery` and from clause had an `Entity`. But now with this change, that check is ignored and the validation is done outside of the `isinstance()` check. Does the validation work for `CompositeQuery`?